pages/linux/gsettings.md

```diff
@@ -0,0 +1,32 @@
+# gsettings
+
+> Query and modify dconf settings with schema validation.
+> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/using_the_desktop_environment_in_rhel_8/configuring-gnome-at-low-level_using-the-desktop-environment-in-rhel-8#using-gsettings-command_configuring-gnome-at-low-level>.
+
+- Display all (non-relocatable) schemas, keys, and values:
+
+`gsettings list-recursively`
+
+- Display all keys and values (default if not set) from one schema:
+
+`gsettings list-recursively {{org.example.schema}}`
+
+- Set the value of a key. Fails if key doesn't exist or value is outside range:
+
+`gsettings set {{org.example.schema}} {{example-key}} {{value}}`
+
+- Un-set a key, so that its schema default value will be used:
+
+`gsettings reset {{org.example.schema}} {{example-key}}`
+
+- Print the value of a key, or schema default if not set (unlike `dconf read`):
+
+`gsetings get {{org.example.schema}} {{example-key}}`
+
+- Display schema-allowed values for a key (helpful with enum keys, less so for others):
```

Comment:
```suggestion
- Display schema-allowed values for a key (helpful with enum keys):
```
Not sure this is needed. Saying that something's useful for one thing implies that it's not as useful for others.